src/main/java/com/connexta/store/config/StoreServiceConfiguration.java

```diff
@@ -32,7 +31,6 @@ public StoreService storeService(
       @NotNull @Named("irmStorageAdaptor") final StorageAdaptor irmStorageAdapter,
       @NotNull @Named("metacardStorageAdaptor") final S3StorageAdaptor metacardStorageAdapter,
       @NotNull final IndexDatasetClient indexDatasetClient,
-      @NotNull final StatusService statusService,
```

Comment:
❓ This was added as part of @ahoffer's poller changes. It wouldn't break the transform request. Did you mean to remove it?

Reply:
I didn't want to mock it in the unit tests. We can add it in once it's actually being used.